frontend/src/components/CreatableSelect.tsx

```diff
@@ -68,17 +65,17 @@ export default function CreatableSelect<T>({
         (optionsLoadable.state === "hasValue" && optionsLoadable.contents.map(toOption)) ||
         undefined
       }
-      onCreateOption={async optionName => {
-        try {
-          setOptionCreating(true);
-          const newOpt = await optionCreator(optionName);
-          if (isDefined(newOpt)) {
-            onChange(newOpt);
-          }
-        } finally {
-          setOptionCreating(false);
-        }
-      }}
+      // onCreateOption={async optionName => {
```

Comment:
Why you removed it?

Reply:
because designers told me that it's unnecessary because the idea was to be able to create new if you need it instantly right now. but now we have these buttons, so they solve this problem.

one more reason it's a little bit difficult to handle this flow: open new dialog, wait for new value from it, then select it as current. so I just discarded it :putya: (I introduced so I discarded)

Reply:
Then remove it, not comment